README.aspnet.md

```diff
@@ -1,3 +1,13 @@
+## Important: Client Firewall Rules Update to Microsoft Container Registry (MCR)
```

Comment:
Please consider fixing up the markdown to be cleaner.

* Redundant multiple lines
* Trailing spaces at the end of lines
* Horizontal rule should have a blank line before it